qupath-core/src/main/java/qupath/lib/images/servers/ImageServerProvider.java

```diff
@@ -170,7 +171,7 @@ public static URI legacyPathToURI(String path) throws IOException {
 
 	private static <T> List<UriImageSupport<T>> getServerBuilders(final Class<T> cls, final String path, String...args) throws IOException {
 		URI uri = legacyPathToURI(path);
-		if ("file".equals(uri.getScheme()) && !new File(uri).exists()) {
+		if ("file".equals(uri.getScheme()) && !Paths.get(uri).toFile().exists()) {
```

Comment:
If you are creating a `File` from a `Path` only to check it exists then I guess using [`Files.exists(path)`](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/nio/file/Files.html#exists(java.nio.file.Path,java.nio.file.LinkOption...)) would make more sense?